regression/goto-analyzer/constant_propagation_09/test.desc

```diff
@@ -1,9 +1,8 @@
-FUTURE
+KNOWNBUG
 main.c
---constants --simplify out.gb
+--constants --verify
```

Comment:
Why has this changed task?

Reply:
For completeness: I have changed the task here as simplification did always take place, but not enough of it to actually turn the assertion into a firm `FALSE`.